homeassistant/components/yandex_transport/manifest.json

```diff
@@ -2,6 +2,5 @@
   "domain": "yandex_transport",
   "name": "Yandex Transport",
   "documentation": "https://www.home-assistant.io/integrations/yandex_transport",
-  "requirements": ["ya_ma==0.3.8"],
```

Comment:
Why do we remove this?

Reply:
This library doesn't work correctly with recent API changes. I replaced it with new async library.